ProjectLighthouse.Servers.Website/Pages/Partials/TwoFactorPartial.cshtml

```diff
@@ -45,7 +40,7 @@
 
 @if (allowBackupCodes)
 {
-<text>
+    <text>
```

Comment:
I see that Rider wants to auto-indent this, so just match the indentation on the closing tag